packages/jest-test-sequencer/src/index.ts

```diff
@@ -219,9 +221,11 @@ export default class TestSequencer {
       if (test != null && !testResult.skipped) {
         const cache = this._getCache(test);
         const perf = testResult.perfStats;
+        const testRuntime =
+          perf.runtime ?? test.duration ?? perf.end - perf.start;
         cache[testResult.testFilePath] = [
-          testResult.numFailingTests ? FAIL : SUCCESS,
-          perf.runtime || 0,
+          testResult.numFailingTests > 0 ? FAIL : SUCCESS,
```

Comment:
not related to the rest of the changes, but I don't like `0` being falsy as the check 😅